ecosystem/nft-metadata-crawler-parser/src/asset_uploader.rs

```diff
@@ -134,6 +137,23 @@ impl AssetUploaderContext {
         for url in urls.urls.clone() {
             let self_clone = self_clone.clone();
             tasks.push(tokio::spawn(async move {
+                let mut conn = self_clone.pool.get().context("Failed to get connection")?;
+                if let Some(model) = NFTMetadataCrawlerURIsQuery::get_by_asset_uri(&mut conn, url.as_ref()) {
```

Comment:
Is there a reason we do get-upload here instead of just uploading again? Does an additional upload fail?

Reply:
Cloudflare returns an error if we upload the same asset url 😦

Reply:
If not, this is okay, just document in a comment that the other approach wasn't possible.